### DISCUSSION TITLE
Official Documentation for Nextcloud-AIO admins

### DISCUSSION THREAD

**Discussion by Conkernel** [nextcloud/all-in-one#5129] [Questions] [answered]
Hi,
I was wondering if it's anyway expected any kind of manual for the AIO version of nextcloud. Comming from Server version, I can see most of the utilities now are docker based, and I'd like to be able to inmerse a little bit more on the nextcloud AIO architecture.

Thanks a lot!!

**Comment by szaimen** [comment, marked as answer]
Hi, most of AIOs documentation is available here: https://github.com/nextcloud/all-in-one (if you scroll down)

Additional documentation is available here: https://github.com/nextcloud/all-in-one/discussions/categories/wiki
